### DISCUSSION TITLE
Entry::query() code completion not working (VS Code)

### DISCUSSION THREAD

**Discussion by adevade** [statamic/cms#5950] [Q&A]
Since the `Entry` facade defines an empty interface as return type, VS Code (with Intelephense) does not know which methods are available.

What could be done to allow code completion? Should I not be using the facade?

![image](https://user-images.githubusercontent.com/1066486/165918110-ed9da9e2-cc90-4c4b-a359-63107a6132f4.png)

![image](https://user-images.githubusercontent.com/1066486/165918920-5009945f-e999-4576-8abb-bf44bd48d39b.png)

**Comment by edalzell** [comment]
Need to PR the type hints into Statamic
